elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1672,27 +1712,13 @@ protected void addInverseRelation(String relationName, Object inverseObj) {
      * @param permission the permission
      * @param resources  the resources
      * @return Filtered set of resources
-     * @deprecated use {@link PersistentResource#filter(Class, Set, boolean)}
      */
-    @Deprecated
     protected static Set<PersistentResource> filter(Class<? extends Annotation> permission,
                                                     Set<PersistentResource> resources) {
-        return filter(permission, resources, false);
-    }
-
-    /**
-     * Filter a set of PersistentResources.
-     *
-     * @param permission the permission
-     * @param resources  the resources
-     * @param skipNew whether or not objects created this transaction should be skipped
-     * @return Filtered set of resources
-     */
-    protected static Set<PersistentResource> filter(Class<? extends Annotation> permission,
-                                                    Set<PersistentResource> resources, boolean skipNew) {
         Set<PersistentResource> filteredSet = new LinkedHashSet<>();
         for (PersistentResource resource : resources) {
             try {
+                boolean skipNew = resource.getRequestScope().isMutatingMultipleEntities();
```

Comment:
This variable name doesn't match what we are checking.  Why not call it isMutation or something like that?